### PR TITLE
[pro update] Copy update on /desktop/developers

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done

- Copy update on /desktop/developers

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/desktop/developers or https://ubuntu-com-11823.demos.haus/desktop/developers
    - Be sure to test on mobile, tablet and desktop screen sizes
- Compare page against [copy doc](https://docs.google.com/document/d/1VJ3xT2ViC-CvL0UmsAUzpYFmvpuqUP4AbpnlPm8tqOA/edit#heading=h.q5r79xcdu7p7)

## Issue / Card

Fixes #https://github.com/canonical-web-and-design/web-squad/issues/5618

### CODE REVIEW

File: templates/desktop/developers.html [hunk 1]
```diff
@@ -26,228 +26,562 @@ <h1>Ubuntu Desktop for&nbsp;Linux developers</h1>
...
+            ) | safe
+          }}
+        </div>
       </div>
     </div>
   </section>
+  <section class="p-strip is-bordered">
+    <div class="row">
+      <div class="col-8">
+        <h2>What's new in Ubuntu Server {{ releases.lts.short_version }} LTS</h2>
```
Comment (anthonydillon) [inline templates/desktop/developers.html:522]: Copy docs says: "What’s new in Ubuntu 22.04?"

File: templates/desktop/developers.html [hunk 2]
```diff
@@ -26,228 +26,562 @@ <h1>Ubuntu Desktop for&nbsp;Linux developers</h1>
...
+        <ul>
+          <li>Linux 5.15 kernel</li>
+          <li>New Ubuntu theme with accent colors, improved integration for snaps and system-wide dark style preference support</li>
+          <li>GNOME Shell and most GNOME libraries updated to the 42 release
+            <ul>
+              <li>Streamlined user interface and workspace transitions</li>
+              <li>Compact and modern on-screen display notifications</li>
+              <li>New screenshot/screen recording interfacetreamlined user interface and workspace transitions</li>
```
Comment (anthonydillon) [inline templates/desktop/developers.html:530]: Copy doc just has: "New screenshot/screen recording interface"

File: templates/desktop/developers.html [hunk 3]
```diff
@@ -26,228 +26,562 @@ <h1>Ubuntu Desktop for&nbsp;Linux developers</h1>
...
-  <section class="p-strip">
-    <div class="row u-equal-height u-vertically-center u-hide--small u-hide--medium">
-      <div class="col-5">
-        {{
-          image(
-            url="https://assets.ubuntu.com/v1/bda34a6e-wordpress-apache-mysql-bundle-trans.svg",
+    <div class="row">
+      <div class="col-6">
+        <h3 class="u-sv2 p-heading--4">Operating systems developers prefer</h3>
+        <p>Multiple choice poll<p>
```
Comment (anthonydillon) [inline templates/desktop/developers.html:488]: I think this can be hidden on small and medium screen as it doesn't make sense without the chart.

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Seems the page is missing the "Trusted manageability with Ubuntu Pro Desktop" section.

**Comment by anthonydillon** [comment]
@minkyngkm looking good. Almost there just a few comments in the code and suggestion in the smaller screens QA.

File: templates/desktop/developers.html (hunk 3 above)
Comment (minkyngkm) [reply templates/desktop/developers.html:499]: Could I hide all in this section?

**Comment by minkyngkm** [comment]
Thanks @anthonydillon for reviewing. The copy doc seems to be updated a bit after I worked but it's been all updated now. Thanks! 👍

**Review by anthonydillon — APPROVED** [review]
LGTM :+1: thanks for the updates Min.

**Comment by minkyngkm** [comment]
Oliver's feedback (21/07/2022) :
- [ ] You've pulled the wrong graph from the source doc for 'open source technologies' we need the one with Ubuntu and it would be good to do a redesign so it's the same style as the 'operating system developers prefer' on the left (ie orange and grey)
- [ ] The logo for Ubuntu Pro Desktop has the old logo, needs updating and maybe have it on the right and a bit bigger so - [ ] it's more like the live version of the page with the bullets on the left
- [ ] It would be good to reformat the 'what's new in Ubuntu 22.04' section to be identical to how it's portrayed on the live site
Missing hyperlinks to ask Ubuntu and mailing lists

**Comment by anthonydillon** [comment]
@minkyngkm is this one good to go?

**Comment by minkyngkm** [comment]
@anthonydillon From what I checked with Oliver today, it is blocked until October 5th (as this page is part of the advantage-pro updates and it was postponed) and there are still some things to change on this page. Oliver said he will round up the feedback in a final, structured way and let us know.

File: templates/desktop/developers.html [hunk 4]
```diff
@@ -86,151 +517,72 @@ <h2>What's new in Ubuntu {{ releases.lts.short_version }}?</h2>
             <li>Compact and modern on-screen display notifications</li>
             <li>New screenshot/screen recording interface</li>
             <li>New power profile selection</li>
-            <li>Improved multi-monitor support</li>
+            <li>mproved multi-monitor support</li>
```
Comment (mtruj013) [inline templates/desktop/developers.html:520]:
```suggestion
            <li>Improved multi-monitor support</li>
```

File: templates/desktop/developers.html [hunk 5]
```diff
@@ -86,151 +517,72 @@ <h2>What's new in Ubuntu {{ releases.lts.short_version }}?</h2>
...
         </li>
         <li class="p-list__item is-ticked">GNOME desktop performance improvements
           <ul>
-            <li>Up to 2X improvement in frame rates with Intel and Raspberry Pi graphics drivers</li>
+            <li>Up to 2x improvement in frame rates with Intel and Raspberry Pi graphics drivers</li>
             <li>Significant improvements for AMD and NVIDIA</li>
           </ul>
         </li>
-        <li class="p-list__item is-ticked">Wayland as the default display server for most systems whilst X11 remains the default for those with an NVIDIA discrete GPU</li>
+        <li class="p-list__item is-ticked">Wayland ais the default display server for most systems whilst X11 remains the default for those with an NVIDIA discrete GPU</li>
```
Comment (mtruj013) [inline templates/desktop/developers.html:529]:
```suggestion
        <li class="p-list__item is-ticked">Wayland as the default display server for most systems whilst X11 remains the default for those with an NVIDIA discrete GPU</li>
```

File: templates/desktop/developers.html [hunk 6]
```diff
@@ -86,151 +517,72 @@ <h2>What's new in Ubuntu {{ releases.lts.short_version }}?</h2>
...
         <li class="p-list__item is-ticked">Latest PulseAudio support for LDAC and AptX bluetooth codecs, HFP bluetooth profiles and bluetooth device battery level reporting</li>
-        <li class="p-list__item is-ticked">Up-to-date toolchains, including Python 3.10, PHP 8.1, Ruby 3.0, Perl 5.34, Go 1.18, GCC 11.2, Rust 1.58.1</li>
-        <li class="p-list__item is-ticked">Core productivity apps LibreOffice 7.3.2, Thunderbird 91.8, and Firefox 99 which is now included as a snap</li>
+        <li class="p-list__item is-ticked">Up-to-date toolchains, including Python 3.10, PHP 8.1, Ruby 3.0, Perl 5.34, Go 1.18, GCC 11.2, Rust 1.58</li>
```
Comment (mtruj013) [inline templates/desktop/developers.html:531]:
```suggestion
        <li class="p-list__item is-ticked">Up-to-date toolchains, including Python 3.10, PHP 8.1, Ruby 3.0, Perl 5.34, Go 1.18, GCC 11.2, Rust 1.58.1</li>
```

File: templates/desktop/developers.html [hunk 7]
```diff
@@ -86,151 +517,72 @@ <h2>What's new in Ubuntu {{ releases.lts.short_version }}?</h2>
...
-    </div>
-  </section>
-
-  <section class="p-strip--suru-accent is-dark">
     <div class="u-fixed-width">
-      <blockquote class="p-pull-quote">
-        <p class="p-pull-quote__quote">We want to be able to deliver the same high-quality experience on Linux as we do on other platforms. Snaps allow us to do just that, by giving us the ability to push the latest features straight to our users, no matter what device or distribution they happen to use.</p>
-        <cite class="p-pull-quote__citation">Jonáš Tajrych, Senior Software Engineer at Skype at Microsoft</cite>
-      </blockquote>
+      <p><a href="{{ releases.lts.release_notes_url }}">Read more in the release notes&nbsp;&rsaquo;</a></p>
```
Comment (mtruj013) [inline templates/desktop/developers.html:542]:
```suggestion
      <p><a href="{{ releases.lts.release_notes_url }}">Read the full release notes&nbsp;&rsaquo;</a></p>
```

**Review by mtruj013 — DISMISSED** [review]
Looks a great @minkyngkm! Just a few copy discrepancies and also the values for the open source data chart are a bit off from what's on the doc

**Comment by mtruj013** [comment]
Oh also, forgot to mention this above, but the "latest toolchains" link routes to this same page which seems a bit weird. I left a [comment](https://docs.google.com/document/d/1VJ3xT2ViC-CvL0UmsAUzpYFmvpuqUP4AbpnlPm8tqOA/edit?disco=AAAAgslIFbo) in the doc asking for clarification

**Review by sowasred2012 — APPROVED** [review]
This LGTM, and @mtruj013's were addressed

**Pull request merged**